knative/integration_test.go

```diff
@@ -0,0 +1,157 @@
+//go:build integration
+// +build integration
+
+package knative_test
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	fn "knative.dev/func"
+	"knative.dev/func/k8s"
+	"knative.dev/func/knative"
+)
+
+func TestIntegration(t *testing.T) {
```

Comment:
This looks great! Looks like this change is great for our coverage. But would it not be better in the long run to have individual tests for deploy, describe and list? I know that means deploying three separate times, so maybe it's overkill.

Reply:
I want to test that all the operation works together.